apollo-federation/src/subgraph/database.rs

```diff
@@ -13,29 +13,21 @@
 
```

Comment:
I think we can remove this whole file. It's only used in a test

Reply:
Are you saying to move the content to where the test is or to also rewrite that test?

Reply:
AFAICT it's only used in a test *that tests this file*, so we can remove this file and the test.